rules/chrome.json

```
@@ -0,0 +1,55 @@
+{
+  "patterns": [
+    "\\b[cC]hrome\\b\\s+\\b[bB]rowser\\b",
+    "\\b[gG]oogle\\b\\s+\\b[cC]hrome\\b"
```

```suggestion
    "\\bchrome\\b\\s+\\bbrowser\\b",
    "\\bgoogle\\b\\s+\\bchrome\\b"
```

The patterns are case-insensitive, so the `[cC]` isn't strictly needed. Alternatively, it could be capitalized for readability:

```suggestion
    "\\bChrome\\b\\s+\\bBrowser\\b",
    "\\bGoogle\\b\\s+\\bChrome\\b"
```

---

On second thought, how about just `\\bchrome\\b`? I'm looking at SystemRequirements for a few other packages that need Chrome, and some of them only write "Chrome" without the Google or Browser part. For example, https://github.com/RLesur/crrri/blob/69c54e657f117b9e30b6a5475604e1e0c6584150/DESCRIPTION#L17

---

I'm ok with just `Chrome`. My original thought was the word `Chrome` was too _common_. But if you're writing `SystemDependencies`, I guess you could be careful.

---

```suggestion
    "\\bchrome\\b"
```